public/index.html

```diff
@@ -6,6 +6,124 @@
   <link rel="stylesheet" href="/app.css">
 </head>
 <body>
+    <div class="navbar">
+        <div class="menu__left">
+          <div class="logo">
+              <i class="fa fa-arrow-left"></i>
+              /
+              <i class="fa fa-tint"></i>
+          </div>
+          <h4 class="name">Stefan Theofilos</h4>
+          <h4 class="site-name">Photography</h4>
+        </div>
+        <div class="menu__right">
+          <h4 class="about">About</h4>
+          <h4 class="work">Work</h4>
+          <h4 class="contact">Contact</h4>
+        </div>
+    </div>
 
+    <div class="main">
+
+<!-- Header -->
+      <div class="header">
+        <figure class="avatar__gram">
+          <img class="avatar" src="http://placecage.com/100/100" alt="">
+        </figure>
+        <i class="fa fa-caret-down"></i>
+        <h3 class="site-title">Landscape</h3>
+      </div>
+
+<!-- Article -->
+      <div class="article">
+        <div class="article__info">
+          <figure class="article__photo-gram">
+            <img class="article__photo" src="photos/cars.JPG" alt="cars">
+          </figure>
+          <div class="article__info-text">
```

Comment:
I think you'll find adding more more grouping to contain the `article__photo-gram` and `article__info-text` as the left side